src/views/ManageList.jsx

```diff
@@ -1,7 +1,78 @@
+import { useState } from 'react';
+import { addItem } from '../api';
+
```

Comment:
I see the `addItem` import here, but it doesn't look like it's being used anywhere. That function is actually that one that submits to Firebase